static/docs/commands-reference/gc.md

```diff
@@ -48,7 +48,7 @@ current branch/commit (unless `-a` or `-T` is specified).
 * `-r`, `--remote` - name of the remote storage to collect unused objects from
 if `-c` option is specified.
 
-* `-j`, `--jobs` - garbage collector parallelism level. Default is
+* `-j`, `--jobs` - garbage collector parallelism level. The default value is
```

Comment:
thanks!!

Reply:
Np. some times I notice small problems like this one in other files and take the liberty to include them in my changes. Should have told you about it directly but forgot - glad you noticed here! Please keep an eye for more of these